### DISCUSSION TITLE
child window

### DISCUSSION THREAD

**Discussion by zoy-l** [tauri-apps/tauri#6653] [Q&A]
Hi, I'm new to Tauri, and I was wondering if Tauri can create a `child window`. This is a barrier to migrating from `Electron` to `Tauri`. Note that not multiple windows, the Electron main window, and child windows can communicate directly(`window.opener`) without the need for an `event`.

**Comment by FabianLars** [comment]
About child windows in general, basically this: https://github.com/tauri-apps/tauri/discussions/5785#discussioncomment-4348980 - cross window communication in tauri is only possible via events right now, no matter if they're "related" or not, and i'm not sure if we can actually change that (apart from adding a nicer api on top of it).

I guess this question would be part of https://github.com/tauri-apps/tauri/issues/1643

**Reply by zoy-l** [reply]
Thank you for your reply, you did a great job :) What I want to express is that there is no need to change the original things, it would be great to create a sub-window and get the window instance.
